lib/mintacoin_web/controllers/accounts_controller.ex

```diff
@@ -0,0 +1,63 @@
+defmodule MintacoinWeb.AccountsController do
+  @moduledoc """
+  This module contains the account endpoints
+  """
+
+  use MintacoinWeb, :controller
+
+  alias Ecto.Changeset
+  alias Mintacoin.{Account, Accounts, Blockchain, Blockchains}
+  alias MintacoinWeb.ErrorView
+
+  @type params :: map()
+  @type conn :: Plug.Conn.t()
+  @type status :: :ok | :created
+  @type template :: String.t()
+  @type response :: Account.t() | String.t() | nil
+  @type error :: :not_found | :bad_request | Changeset.t()
+  @type message_error :: String.t()
+
+  action_fallback MintacoinWeb.FallbackController
+
+  @spec create(conn :: conn(), params :: params()) :: conn() | {:error, error()}
+  def create(%{assigns: %{network: network}} = conn, %{"blockchain" => blockchain}) do
+    case Blockchains.retrieve(blockchain, network) do
+      {:ok, %Blockchain{} = blockchain} ->
+        blockchain |> Accounts.create() |> handle_response(conn, :created, "account.json")
+
+      {:ok, nil} ->
+        handle_error_response(conn, "blockchain not found")
+    end
```

Comment:
@jhonatan-kmt IMO a better approach would be to retrieve the blockchain in a private function for example, in order to only have in the endpoint functions a pipe:
 ```suggestion
    blockchain
    |> fetch_blockchain(network)
    |> Accounts.create()
    |> handle_response(conn, :created, "account.json")
```